jax/_src/cloud_tpu_init.py

```diff
@@ -70,3 +71,11 @@ def cloud_tpu_init() -> None:
 
   if 'JAX_USE_PJRT_C_API_ON_TPU' not in os.environ:
     os.environ['JAX_USE_PJRT_C_API_ON_TPU'] = 'true'
+
+  use_pjrt_c_api = os.environ['JAX_USE_PJRT_C_API_ON_TPU']
+  if use_pjrt_c_api in ("false", "0"):
+    warnings.warn(
+        f"JAX_USE_PJRT_C_API_ON_TPU={use_pjrt_c_api} will no longer be "
+        "supported in an upcoming future release. Please file an issue at "
+        "https://github.com/google/jax/issues if you are still using this "
```

Comment:
"need" perhaps?

The first action should be to "try turning it off".

Reply:
Good point, updated. FWIW I don't think anyone is using this setting.